quiz/web/src/modules/common/components/Start/Start.tsx

```diff
@@ -0,0 +1,69 @@
+import Box from "@mui/material/Box";
```

Comment:
```quiz/web/src/modules/common/components/Start/Start.tsx``` is incorrect start is a view, therfore it should obtain it's own module. ```quiz/web/src/modules/Start/Start.tsx```

Reply:
https://alexkondov.com/tao-of-react/
Atleast few interesting points here

Reply:
fixed